### PR TITLE
English review of lines 100-200 in FAQ

### PR DESCRIPTION
**Pull request by Rashadse**
This is fun because i am learning about ring and programming in general  :)

Issues:
i couldn't change lines 115 and 116 because it wasn't clear to me what they mean. They might need an extra eye and maybe you can remove the text inside the parenthesis

### CODE REVIEW

File: docs/source/faq.txt [hunk 1]
```diff
@@ -102,19 +102,20 @@ new paradigm through simple prototypes during the last years and we will integra
...
-comes first is what you want. When you type "Print : " + 5 , The String comes first then 5
-will be converted to a String. while when you type 5 + "10" The number comes first so "10"
-will be converted to 10. This help a lot to quickly convert between numbers and strings using
+Because it's faster and more natural, and this is important for the language's goals. One
+of the rules is: the data type at the beginning affects the final result. 
+For example, when you type "Print : " + 5 , The String comes first, so 5
+will be converted to a String. While when you type 5 + "10" The number comes first so "10"
+will be converted to 10. This helps a lot to quickly convert between numbers and strings using
 the same operator. If you want to prevent conversion (Write code that prevent conversion) In
 these cases you will notice that what you are writing is less code (And can be removed).
```
Comment (Rashadse) [inline docs/source/faq.txt]: (can be removed)?

File: docs/source/faq.txt [hunk 2]
```diff
@@ -102,19 +102,20 @@ new paradigm through simple prototypes during the last years and we will integra
...
-Because it's more Natural, and Faster and this is important for the language goals. What
-comes first is what you want. When you type "Print : " + 5 , The String comes first then 5
-will be converted to a String. while when you type 5 + "10" The number comes first so "10"
-will be converted to 10. This help a lot to quickly convert between numbers and strings using
+Because it's faster and more natural, and this is important for the language's goals. One
+of the rules is: the data type at the beginning affects the final result. 
+For example, when you type "Print : " + 5 , The String comes first, so 5
+will be converted to a String. While when you type 5 + "10" The number comes first so "10"
+will be converted to 10. This helps a lot to quickly convert between numbers and strings using
 the same operator. If you want to prevent conversion (Write code that prevent conversion) In
```
Comment (Rashadse) [inline docs/source/faq.txt]: (Write code that prevent conversion)?

**Comment by MahmoudFayed** [comment]
I revised them, They just provide more details.

Nice update, Keep up the good work

File: docs/source/faq.txt (hunk 1 above)
Comment (MahmoudFayed) [reply docs/source/faq.txt]: Yes

File: docs/source/faq.txt (hunk 2 above)
Comment (MahmoudFayed) [reply docs/source/faq.txt]: You can remove it.

**Pull request merged**